.clang-tidy

```diff
@@ -1,11 +1,19 @@
 ---
-Checks:          'google-readability-casting,modernize-use-default-member-init,modernize-use-using,readability-redundant-member-init'
+Checks:          'google-readability-casting,modernize-deprecated-headers,modernize-loop-convert,modernize-use-auto,modernize-use-default-member-init,modernize-use-using,readability-else-after-return,readability-redundant-member-init,readability-static-accessed-through-instance,readability-redundant-string-cstr,readability-uppercase-literal-suffix'
```

Comment:
I think this is adding too much. Some of these are not helpful, IMHO.
The modernize transformations should not be applied to unit tests without careful hand-review.

`readability-static-accessed-through-instance` is particularly incorrect, I have to say.